pallets/eth-bridge/src/lib.rs

```diff
@@ -749,6 +749,9 @@ pub mod pallet {
                     .map_err(|_| Error::<T>::QuotaReachedForAdditionalEvents)?;
                     Self::deposit_event(Event::<T>::AdditionalEventQueued { transaction_hash });
                 },
+                AdminSettings::RestartEventDiscoveryOnRange => {
+                    let _ = EthereumEvents::<T>::clear(10, None);
```

Comment:
I am assuming 10 is how many keys we want to clear? If so, can we increase it in case we have more collators in the future?